docs/runbooks/HCOGoldenImageWithNoArchitectureAnnotation.md

```diff
@@ -0,0 +1,87 @@
+# HCOGoldenImageWithNoArchitectureAnnotation
+
+## Meaning
+
+When running on a heterogeneous cluster, a cluster with nodes of different
+architectures, the DataImportCronTemplates (DICTs; also known as golden
+images), in the hyperconverged cluster operator (HCO) should be annotated with
+the `ssp.kubevirt.io/dict.architectures` annotation, where the value is the
+list of the architectures supported by the image, that is defined in each DICT.
```

Comment:
Maybe add that this alert is based on the feature that was enabled?
So that users that already have DICTs will understand why they suddenly start to get these alert